docs/src/installation.md

```diff
@@ -15,8 +20,10 @@ ARCH=$(uname -m)
 RELEASE="ray-2.5.1+1"
 pip install -U "ray[default] @ https://github.com/beacon-biosignals/ray/releases/download/$RELEASE/${RELEASE%+*}-${PYTHON}-${OS}_${ARCH}.whl" "pydantic<2"
 
-# Install the Julia packages "ray_julia_jll" and "Ray"
-TAG="v0.1.0" julia -e 'using Pkg; Pkg.add(PackageSpec(url="https://github.com/beacon-biosignals/Ray.jl", rev=ENV["TAG"]))'
+ray start --head && ray stop
```

Comment:
Works around this issue with brand new installations:
```
julia> Ray.init()
ERROR: IOError: realpath("/tmp/ray/session_latest/logs"): no such file or directory (ENOENT)
```
Manually creating that directory causes: 
```
julia> Ray.init()
ERROR: SystemError: opening file "/tmp/ray/session_latest/logs/raylet.out": No such file or directory
```
and creating that file still causes another issue:
```
julia> Ray.init()
ERROR: Unable to locate agent process information
```
Even `ray start --head` isn't fully reliable but seems to resolve a few initial issues